FrontEndApp/v1-react/src/Components/MainApp/Step2.js

```diff
@@ -2927,7 +2924,10 @@ function Step2() {
     };
     const res = await HomeService.download_code(token, data);
     if (res.status === 200) {
-      fileDownload(res.data, "test.py");
+      //check if a valid name exists else give the default name as "output"
```

Comment:
I personally think this comment isn't necessary, what you're trying to do is pretty much clear from the code :) 
```suggestion
```

Reply:
Deleted it. Thanks for the suggestion :)